pkg/apis/servicecatalog/validation/serviceclass_test.go

```diff
@@ -24,171 +24,124 @@ import (
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
+func validServiceClass() *servicecatalog.ServiceClass {
```

Comment:
Not sure what you mean - i think the default fuzzer is fine for this field. Were you asking if we need a special fuzzer for these types?